Uno.Gallery/Uno.Gallery.Wasm/Uno.Gallery.Wasm.csproj

```diff
@@ -108,9 +108,9 @@
 		<PackageReference Include="Uno.Toolkit.UI" Version="2.3.0" />
 		<PackageReference Include="Uno.Toolkit.UI.Cupertino" Version="2.3.0" />
 		<PackageReference Include="Uno.Toolkit.UI.Material" Version="2.3.0" />
-		<PackageReference Include="Uno.UI.Lottie" Version="4.8.0-dev.458" />
-		<PackageReference Include="Uno.UI.WebAssembly" Version="4.8.0-dev.458" />
-		<PackageReference Include="Uno.UI.RemoteControl" Version="4.8.0-dev.458" Condition="'$(Configuration)'=='Debug'" />
+		<PackageReference Include="Uno.UI.Lottie" Version="4.9.0-dev.890" />
+		<PackageReference Include="Uno.UI.WebAssembly" Version="4.9.0-dev.890" />
+		<PackageReference Include="Uno.UI.RemoteControl" Version="4.9.0-dev.890" Condition="'$(Configuration)'=='Debug'" />
```

Comment:
For wasm there's a new package that needs to be included which is not yet published (will do soon).

Reply:
Yes @jeromelaban I am waiting on [Uno 4.9.0-dev.917](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=77971&view=results) for WASM.